protocol/common/message.go

```diff
@@ -49,6 +49,12 @@ type QuotedMessage struct {
 
 type CommandState int
 
+type MuteParams struct {
```

Comment:
There's a namespace for API parameters `protocol/requests`, we should move this there